### PR TITLE
Make function File.copy accept parameters

### PR DESCRIPTION
**Pull request by StephanMeijer**
Useful for example for giving new parent id's

### CODE REVIEW

File: lib/google_drive/file.rb [hunk 1]
```diff
@@ -213,9 +213,9 @@ def rename(title)
     alias title= rename
 
     # Creates copy of this file with the given title.
-    def copy(title)
+    def copy(title, params = {})
```
Comment (gimite) [inline lib/google_drive/file.rb]: Thanks! Looks useful. But it's a bit confusing to call it `params`. In this library, `params` usually refer to the third argument of copy_file e.g., `fields` and `supports_team_drives`. How about calling it `file_properties` instead?

Comment (StephanMeijer) [reply lib/google_drive/file.rb]: I am totally fine with that! Renaming!

**Comment by StephanMeijer** [comment]
Renaming is done @gimite :)

**Comment by gimite** [comment]
Thanks for the pull request! Merged.

**Pull request merged**